config/locales/hyrax.zh.yml

```diff
@@ -8,6 +8,7 @@ zh:
       messages:
         conflict: 您的更改无法保存，因为另一用户(或后台作业）在您开始编辑后更新了%{model}。 请确保所有文件附件已成功完成之后再试一次。这已经是最新刷新后保存的副本%{model}
   blacklight:
+    application_name: must be here for features page to open
```

Comment:
I think we should move this to https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/templates/config/locales/hyrax.zh.yml, because that file gets generated into the application, and it makes it easier for people to change it.  Long term it would be great to get a Chinese translation into Blacklight.